src/node.h

```diff
@@ -375,8 +375,6 @@ NODE_DEPRECATED("Use WinapiErrnoException(isolate, ...)",
 })
 #endif
 
-const char *signo_string(int errorno);
```

Comment:
The contents of this file may be used in add-ons, so I think removing something here would have to qualify as semver-major (I may be wrong about that).